### PR TITLE
Upgrade to napi-rs@0.3 cli

### PR DESCRIPTION
**Pull request by Brooooooklyn**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,15 +33,20 @@
   "cpu": [
     "x64"
   ],
+  "napi": {
```
Comment (kdy1) [inline package.json:36]: Seems like a very clever trick.

**Review by kdy1 — APPROVED** [review]
Thanks!

**Comment by Brooooooklyn** [comment]
Don't forget to rerun `yarn` to install latest `napi-rs` before `npm version patch && git push --follow-tags`, `napi-rs` cli will do things while version bump happening.

**Comment by kdy1** [comment]
@Brooooooklyn Thank you for the notice! 
I'll do it before publising a new version.

**Pull request merged**